chef/cookbooks/ceph/libraries/default.rb

```diff
@@ -93,7 +93,9 @@ def get_mon_addresses()
 
     mons += get_mon_nodes()
     if is_crowbar?
-      mon_ips = mons.map { |node| Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin").address }
+      mon_ips = mons.map do |node|
+        Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "ceph_client").address
```

Comment:
Good thought. And, actually, making this a bit smarter with a fallback to "admin" might fix the upgrade problem.

The purpose of the separate network is to not have ceph client traffic go over the admin network, because the assumption is that the admin network will usually be crap (1GbE). Additionally, for storage-only deploys, ceph clients probably don't *want* to be on the crowbar admin network with its DHCP server.